packages/vue/src/components/atoms/SfImage/SfImage.stories.js

```diff
@@ -5,6 +5,7 @@ import {
   boolean,
   number,
   object,
+  array
```

Comment:
@AdamPawlinski use Object knobs instead of Array. Object is most UX friendly IMO ;)

Reply:
of course, I removed that.